builder/Dockerfile.linux

```diff
@@ -20,11 +20,6 @@ RUN go build ./cmd/aks-periscope
 # Runner
 FROM alpine
 
-RUN apk --no-cache add ca-certificates curl openssl bash
```

Comment:
💡 we checked and this seems good, we dint find any code with curl dependency et. al. hence removal of this seems justifiable, if we find something we can always add it back again.

Reply:
Also, just as note, the default of `alpine` shell is `ash`